index.html

```diff
@@ -279,8 +279,16 @@ <h3>JsonSchemaCredential</h3>
                of a verifiable credential is <code>https://www.w3.org/ns/credentials#JsonSchema</code>.
             </p>
             <p>
-               The specification version of [[JSON-Schema]] can be any version noted in the section
-               on <a href="#json-schema-specifications">JSON Schema Specifications</a>.
+               When defining a credential schema using the <code>JsonSchemaCredential</code> type,
+               the credential's <code>credentialSubject</code> MUST use the <a href="#jsonschema">JsonSchema</a> 
+               <code>type</code> value and make use of the <code>jsonSchema</code> property, as exemplified below.
+               The <code>credentialSubject</code> MAY have an <code>id</code> property, and if present, 
+               the value MUST match the <code>$id</code> property in the <code>jsonSchema</code> property.
```

Comment:
`my_fun_id` is not a URL and so this statement is at odds with the VCDM rules for identifiers. You may want to highlight that if it MUST match the `$id` property, then it must also be the case that that property's value be a URL. I suspect that may prohibit some existing schemas from being able to be used without changing the IDs -- so maybe you don't want to go this way at all, i.e., don't create this linkage. It also creates a situation where the ID identifies two different things: the descriptor for the JSON schema -- and the JSON schema itself.

Reply:
Afaik, `@id` is JSON-LD and `$id` in JSON Schema are compatible, and both express URLs, either absolute or relative... @decentralgabe is it possible for a spec legal `$id` to not be a valid JSON-LD `@id` ?

Reply:
I'm not certain. I'll take an action to dig into the spec a bit deeper and update this example. Thanks for the callouts.

Reply:
When a URL is relative, it's relative to the base URL of the document. There is no base URL for a VC because its a portable document (issuer gives it to a holder, holder gives it to a verifier). Don't use relative URLs.

Reply:
@dlongley we already have this guidance in the spec https://github.com/w3c/vc-json-schema/blob/main/index.html#L295

Reply:
have updated, should be fine now